Server/rainbow-server/Sources/Application/Model/ScoreEntry.swift

```diff
@@ -21,5 +21,6 @@ struct ScoreEntry: Codable {
     var deviceIdentifier: String?
     var avatarImage: Data?
     var avatarURL: String?
-    var objects: [ObjectEntry]?    
+    var objects: [ObjectEntry]?
+    var totalTime: Double?
```

Comment:
I could be persuaded otherwise, but I am not sure if we need this totalTime variable because we can easily calculate it from `startDate` and `finishDate`.

Reply:
I needed this for the server side to sort the array